src/GraphQL.Tests/Bugs/Bug_nested_type_names_dont_conflict.cs

```diff
@@ -4,7 +4,6 @@
 
 namespace GraphQL.Tests.Bugs
 {
-    [Collection("Sequential_UseDeclaringTypeNames")]
```

Comment:
It does not work. From time to time, some tests fail due to `GlobalSwitches.UseDeclaringTypeNames` setting during parallel execution. This can be avoided only performing most tests in serial i.e. `Collection` attribute should be added to practically each schema test. So it's easier to restart tests.